### PR TITLE
translate configure_upgrade_etcd

### PR DESCRIPTION
**Pull request by hwdef**

### CODE REVIEW

File: content/zh/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 1]
```diff
@@ -67,17 +67,17 @@ content_template: templates/task
...
 -->
 ## 资源要求
 
 使用有限的资源运行 etcd 只适合测试目的。为了在生产中部署，需要先进的硬件配置。在生产中部署 etcd 之前，请查看[所需资源参考文档](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/hardware.md#example-hardware-configurations)。
 
-## 启动 Kubernetes API 服务器
+## 启动 etcd 集群
 
-本节介绍如何在 deployment 中使用 etcd 集群启动 Kubernetes API 服务器。
+本节介绍如何启动单节点和多节点etcd集群。
```
Comment (xichengliudui) [inline content/zh/docs/tasks/administer-cluster/configure-upgrade-etcd.md]:
```suggestion
本节介绍如何启动单节点和多节点 etcd 集群。
```

File: content/zh/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 2]
```diff
@@ -451,4 +451,30 @@ storage backend, please consult the [Kubernetes v1.12 etcd cluster upgrade docs]
...
+
+Because the fix requires gRPC dependency upgrade (to v1.23.0), downstream Kubernetes [did not backport etcd upgrades](https://github.com/kubernetes/kubernetes/issues/72102#issuecomment-526645978). Which means the [etcd fix in kube-apiserver](https://github.com/etcd-io/etcd/pull/10911/commits/db61ee106ca9363ba3f188ecf27d1a8843da33ab) is only available from Kubernetes 1.16.
+
+To urgently fix this bug for Kubernetes 1.15 or earlier, build a custom kube-apiserver. You can make local changes to [`vendor/google.golang.org/grpc/credentials/credentials.go`](https://github.com/kubernetes/kubernetes/blob/7b85be021cd2943167cd3d6b7020f44735d9d90b/vendor/google.golang.org/grpc/credentials/credentials.go#L135) with [etcd@db61ee106](https://github.com/etcd-io/etcd/pull/10911/commits/db61ee106ca9363ba3f188ecf27d1a8843da33ab).
+
+See ["kube-apiserver 1.13.x refuses to work when first etcd-server is not available"](https://github.com/kubernetes/kubernetes/issues/72102). 
+-->
+
+## 已知问题：具有安全端点的 etcd 客户端平衡器
```
Comment (xichengliudui) [inline content/zh/docs/tasks/administer-cluster/configure-upgrade-etcd.md]: 平衡 -> 均衡

**Comment by hwdef** [comment]
@xichengliudui 
Thanks for your suggestion!

**Pull request merged**